src/services/services.ts

```diff
@@ -9,8 +9,26 @@ import { ProjectService } from "./project-service/project-service";
 import { LinkService } from "./link-service/link-service";
 import { FavoriteService } from "./favorite-service/favorite-service";
 
-export const authService = new AuthService(authClient, apiClient);
-export const groupsService = new GroupsService(apiClient);
-export const projectService = new ProjectService(apiClient);
-export const linkService = new LinkService(apiClient);
-export const favoriteService = new FavoriteService(linkService, groupsService);
+export interface ServiceRepository {
+    authService: AuthService;
+    groupsService: GroupsService;
+    projectService: ProjectService;
+    linkService: LinkService;
+    favoriteService: FavoriteService;
+}
+
+export const createServices = (): ServiceRepository => {
```

Comment:
I think we could pass base url as a param here.  Or axios instance (serverApi). We would be able to remove  imports from common/api then. This would make more readable, how to configure services module.

Reply:
hmm, might be a good idea :)